libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/SummaryBottomSheet.java

```diff
@@ -40,7 +45,10 @@
   private ProgressBar rerouteProgressBar;
   private boolean isRerouting;
   private Locale locale;
-  private @NavigationUnitType.UnitType int unitType = NavigationUnitType.NONE_SPECIFIED;
+  @NavigationUnitType.UnitType
+  private int unitType = NavigationUnitType.NONE_SPECIFIED;
```

Comment:
I think it makes sense to align how both the `unitType` and `timeFormatType` are both set to `NONE_SPECIFIED` initially.  Currently the time format is set in the `NavigationViewOptions#builder()`, while `unitType` is set here in `SummaryBottomSheet`

Reply:
Actually, I went ahead and removed the `unitType` initialization here. It's not necessary because we're already initializing it in `MapboxNavigationOptions#builder()` 👀 https://github.com/mapbox/mapbox-navigation-android/blob/2975aea4cbe359bc5916c91fcc18e629f2fdf045/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationOptions.java#L125